docs/reference/objects/package/product.md

```diff
@@ -5,6 +5,7 @@ parent: Package Step
 ---
 
 # Package Product
+
```

Comment:
nit: adding these spaces causes the labels to sit below the title instead of alongside them as per the rest of the docs
![CleanShot 2024-04-23 at 15 21 35](https://github.com/easolhq/easolhq.github.io/assets/48216064/574ffa8f-7e52-43c4-b719-2dc600f27f75)
![CleanShot 2024-04-23 at 15 21 49](https://github.com/easolhq/easolhq.github.io/assets/48216064/606aac2e-d78b-44f9-96a3-efb784776f03)

Reply:
Damn editor 👍 have fixed